pkg/auth/auth0.go

```diff
@@ -0,0 +1,51 @@
+package auth
+
+import (
+	"fmt"
+	"net/http"
+
+	json2 "github.com/consensys/quorum-key-manager/pkg/json"
+	"github.com/consensys/quorum-key-manager/pkg/tls/certificate"
+)
+
+const (
+	Auth0IssuerServerDomain = "auth0.com"
+)
+
+type JWKsResponse struct {
```

Comment:
I believe we can use `https://github.com/square/go-jose` for types and handling of certificates. Check `&jose.JSONWebKeySet{}`.

Reply:
Replaced by jose.JSONWebKeySet{}.